app/models/post.rb

```diff
@@ -0,0 +1,7 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments, foreign_key: :post_id
+  has_many :likes
+
+  scope :recent_three, ->(aut) { where (aut == :author_id).limit(3).order(created_at: :desc) }
```

Comment:
- [ ] Here, you should implement a method that returns the 5 most recent comments for a given post.
you can implement it as below:
```ruby
def recent_comments
  comments.order('created_at Desc').limit(5)
end
```